material-overrides/assets/stylesheets/home.css

```diff
@@ -99,7 +95,7 @@
 .feature-divider {
   width: 80%;
   height: 1px;
-  background-color: var(--light);
+  background-color: var(--divider-color);
```

Comment:
This is the only place this variable is used. The color is the same as the font. Let's just use `--md-default-fg-color` instead and get rid of this variable